Docker/supervisord/adaguc-pgbouncer.conf

```diff
@@ -9,6 +9,17 @@ stdout_logfile_maxbytes=0
 redirect_stderr=true
 command=/adaguc/start.sh
 
+[program:adagucbin]
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+redirect_stderr=true
+environment=
+    ADAGUC_FORK=1,
+    ADAGUC_PATH=/adaguc/adaguc-server-master/,
+    ADAGUC_TMP=/tmp,
+    ADAGUC_CONFIG=/adaguc/adaguc-server-master/python/lib/adaguc/adaguc-server-config-python-postgres.xml
+command=/adaguc/adaguc-server-master/bin/adagucserver
```

Comment:
We need to discuss if it would be a good solution to start this executable once in the python code. The reasoning is that the python code has logic to calculate the environment variables like ADAGUC_ONLINERESOURCE and others.